challenge.md

```diff
@@ -5,19 +5,34 @@ permalink: /challenge.html
 ref: challenge
 ---
 
-### QALD - 10 Challenge
+# 10th Question Answering over Linked Data (QALD) Challenge @ ESWC 2022
 
-This year's *Question Answering over Linked Data* challenge, better known as *QALD*, focuses on **Question Answering over Wikidata**. This will force QA systems to adapt to a new graph as QALD has been DBpedia-centric so far. We will have a novel sub-task on low-resource language QALD to drive accessibility in these languages forward. In particular, QA systems over multilingual KGs widen the access to the Web of Data to lay users. Thus, QA systems increase the accessibility of the Web in general.
+This year's Question Answering over Linked Data challenge has **2 tasks** and focuses on **Question Answering over Wikidata**. This will force QA systems to adapt to a new graph as QALD has been DBpedia-centric so far. 
 
-We also introduce a **new task on comparative QA**, where the goal is to efficiently answer factual multilingual comparative questions using knowledge basesComparative QA has a wide range of use cases, such as academic search engines or e-commerce.
+For QALD-10 task 1, we will have a novel sub-task on **low-resource language QALD (Russian, Chinese, German, English) over Wikidata** to drive accessibility in these languages forward. In particular, QA systems over multilingual KGs widen the access to the Web of Data to lay users. Thus, QA systems increase the accessibility of the Web in general.
```

Comment:
> a novel sub-task on **low-resource language QALD (Russian, Chinese, German, English) over Wikidata**

Are these ones low-resource? I'd rather say: "a novel sub-task on **multilingual QALD (Russian, Chinese, German, English) over Wikidata**"

Reply:
the low-resource argument is a carry-over from the submitted proposal.  so, technically I agree. One could discuss at the conference whether Chinese, Russian and German are low-resource w.r.t. KGQA datasets